### DISCUSSION TITLE
Mock date in tests error

### DISCUSSION THREAD

**Discussion by steveljko** [vitest-dev/vitest#619] [General]
I tried to mock date inside test, but i got this errror:

```
TypeError: __vite_ssr_import_0__.vi.mockCurrentDate is not a function
 ❯ test.test.ts:13:7
     11| 
     12|   it('should go to next month', async () => {
     13|     vi.mockCurrentDate(new Date());
       |       ^
     14| 
     15|     createComponent();
```

VSCode error:
`Property 'mockCurrentDate' does not exist on type 'VitestUtils'.ts(2339)`

I followed official Vitest documentation, but nothing seems to resolve this error.

**Comment by Demivan** [comment]
Date mocking was changed recently https://github.com/vitest-dev/vitest/pull/531. Use `setSystemTime` to mock dates. Docs were not redeployed yet.
